data.h

```diff
@@ -114,6 +114,34 @@ namespace con {
     refineMean(mean, data);
   }
 
+  void readKaggle(vector<Sample> *data, const int offset, const int batchSize) {
+	  data->clear();
+	  std::string fileName = "data/kaggle-test_batch.bin";
+		ifstream input(fileName.c_str(), ifstream::binary);
```

Comment:
No, i am reading the data for each batch independently. Because reading the whole file needs (8 * 3072 + 4) * 300,000 bytes = ~7.4 gb.